core/common/config/config.go

```diff
@@ -474,6 +478,21 @@ func newP2pConfigWithDefault() P2PConfig {
 	}
 }
 
+func GetHostIp() string {
```

Comment:
General methods are recommended in common/util.go.

Reply:
Done, move to common/util.go, and add a unit test.